### PR TITLE
Test script for PSA plugin, Typo fixing and config.ini update

### PR DESCRIPTION
**Pull request by mattbox**

### CODE REVIEW

File: onebot/plugins/acl.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ class ACLPlugin(object):
 
     Configuration settings:
         - ``superadmin``: username to store in the database with
-        ``all_permissions`. username depends on mask chosen.
+        -``all_permissions`: username depends on mask chosen
```
Comment (thomwiggers) [inline onebot/plugins/acl.py]: This change isn't correct (and I'm also spotting a typo with the `)

File: onebot/plugins/psa.py [hunk 2]
```diff
@@ -29,5 +29,8 @@ def psa(self, mask, target, args):
 
             %%psa <message>...
         """
+        msg = ' '.join(args['<message>'] or [])
```
Comment (thomwiggers) [inline onebot/plugins/psa.py]: If you're going to do `or []` I'd rather you return an error message: sending an empty PSA seems silly.

File: docs/config.ini [hunk 3]
```diff
@@ -2,10 +2,14 @@
 nick=OneBotTest
 username=TwoBot
 host=my.irc.network
+
 port=6697
 ssl=true
+ssl_verify = CERT_NONE
```
Comment (thomwiggers) [inline docs/config.ini]: I'd rather have this be a comment: I don't like having insecure behaviour in examples turned on by default.

File: docs/config.ini [hunk 4]
```diff
@@ -30,7 +34,6 @@ guard=onebot.plugins.acl.user_based_policy
 identify_by = mask
 
 [onebot.plugins.acl]
-# Pre-seed acl
```
Comment (thomwiggers) [inline docs/config.ini]: why remove this?

Comment (mattbox) [inline docs/config.ini]: I founded pre-seeding acl wasn't necessary when the plugin was added to the includes list

Comment (mattbox) [inline docs/config.ini]: Oh, okay I understand what you mean. It make sense now. I'll change it back and let you clarify if you want.

**Pull request closed without merging**